tests/contrib/test_kmeans.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+import torch
+
+import kornia
+from kornia.testing import BaseTester
```

Comment:
```suggestion
from testing.base import BaseTester
```

Then you can remove the unnecessary tests, like `test_gradcheck` -- it isn't a abstract class anymore 😄